projectdo

```diff
@@ -287,6 +287,20 @@ try_latex() {
   fi
 }
 
+# Any / Generic
+
+try_build_script() {
+  if [ "$ACTION" = build ]; then
+    if [ -f build.sh ]; then
+      execute "sh -c build.sh"
+      exit
+    else
+      echo "Did not find build.sh script"
+      exit
+    fi
```

Comment:
I think this `else` branch could be removed. It will echo and exit every time projectdo gets to to `try_build_script` and we will never get to the generic `nothing_found` function.